common/user/index.ts

```diff
@@ -221,6 +222,7 @@ export async function queryUser<Select extends UserSelect>(user: User, select: S
 export async function updateUser(userId: string, { email }: Partial<Pick<User, 'email'>>) {
     const user = await getUser(userId, /* active */ true);
     if (user.studentId) {
+        await updateZoomUser((email = validateEmail(email)));
```

Comment:
I think email = validateEmail(email); should always be done in this function, I'd suggest to move it into the first line